src/uct/ib/rdmacm/rdmacm_cm.c

```diff
@@ -113,6 +115,254 @@ void uct_rdmacm_cm_cqs_cleanup(uct_rdmacm_cm_t *cm)
     kh_destroy_inplace(uct_rdmacm_cm_cqs, &cm->cqs);
 }
 
+static ucs_status_t
+uct_rdmacm_cm_query_reserved_qpn_cap(struct ibv_context *verbs,
+                                     struct uct_rdmacm_cm_reserved_qpn_context *ctx)
```

Comment:
not addressed

Reply:
@evgeny-leksikov Failed posting the comment due to network issue. --> Yes, I think so. But I checked the exist functions, none of them make the arg as const, such as uct_rdmacm_cm_get_cq(uct_rdmacm_cm_t *cm, struct ibv_context *verbs, struct ibv_cq **cq_p),  mlx5dv_devx_obj_create(struct ibv_context *context, const void *in, size_t inlen, void *out, size_t outlen).
To align with them, I keep it without const. Let me know please if you feel it's important to change to const.

Reply:
let's add const where it's possible especially in new code, some existing code in UCT/rdmacm pass in args without const because rdma API does this way, so the reason is to avoid type cast inside

Reply:
@evgeny-leksikov Just noticed that once changed verbs parameters to const then would have compile error. The RDMA-CORE APIs finally using the verbs paramters are not defined as const type causes this kind of error. I think this is the same as you mentioned "some existing code in UCT/rdmacm pass in args without const because rdma API does this way, so the reason is to avoid type cast inside".